plugins/networking/ua_network_pubsub_ethernet.c

```diff
@@ -5,15 +5,15 @@
  *   Copyright 2018 (c) Kontron Europe GmbH (Author: Rudolf Hoyler)
  */
 
-#include "ua_plugin_network.h"
-#include "ua_log_stdout.h"
-#include "ua_util.h"
+#include "../../include/ua_plugin_network.h"
```

Comment:
Relative paths should be avoided for includes

Reply:
woops. trusted my ide a bit too much doing the refactor. will change.